apps/dashboard/src/@/components/tx-button/MismatchButton.tsx

```diff
@@ -267,14 +267,12 @@ export const MismatchButton = forwardRef<
             />
           )}
 
-          {dialog === "pay" && account && (
-            <CheckoutWidget
+          {dialog === "pay" && (
+            <BuyWidget
               className="!w-full"
               client={props.client}
               chain={txChain}
               amount="0.01"
```

Comment:
No title / button label here ? Feel like we should customize it for this flow no?

Reply:
Its Buy by default - seems good enough?

Reply:
Yeah i Guess it's generic at this level. Let's try it